docs/changelog.rst

```diff
@@ -51,6 +51,7 @@ Other new things:
   yes`` in your configuration to enable.
 * :doc:`/plugins/fetchart`: A new option to change cover art format. Useful for
   DAPs that do not support some image formats.
+* :doc:`/dev/library`: Add property bitrate_mode.
```

Comment:
We generally only use these prefixes in the changelog for plugins. Also, you might want to surround the name of the new field in backticks to make it render nicely.

Reply:
Okay, I have removed the prefix and added the backticks